app/models/user.rb

```diff
@@ -5,10 +5,16 @@ class User < ActiveRecord::Base
 
   has_many :recommended_items
   has_many :television_shows, through: :recommended_items
-
   has_many :following_relationships,
     class_name: "Relationship", foreign_key: :follower_id
   has_many :following, through: :following_relationships, source: :followed
+  has_many :follower_relationships,
+    class_name: "Relationship", foreign_key: :followed_id
+  has_many :followers, through: :follower_relationships, foreign_key: :follower
+
+  def self.recommended_to_follow(except:)
```

Comment:
Well, it's people you should follow, not people who should follow you.

Since "user.followers" is "people who follow you", I'm worried "recommended followers" is confusing.

Sent from my iPhone

> On Jan 24, 2016, at 7:06 AM, Edward Loveall <notifications@github.com> wrote:
> 
> In app/models/user.rb:
> 
> >    has_many :following_relationships,
> >      class_name: "Relationship", foreign_key: :follower_id
> >    has_many :following, through: :following_relationships, source: :followed
> > +  has_many :follower_relationships,
> > +    class_name: "Relationship", foreign_key: :followed_id
> > +  has_many :followers, through: :follower_relationships, foreign_key: :follower
> > +
> > +  def self.recommended_to_follow(except:)
> follower_suggestions?
> 
> —
> Reply to this email directly or view it on GitHub.
>